doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -276,34 +275,66 @@ At a high level, you can compare these concepts as follows:
+
+- Ray Data can utilize all resources in the Ray cluster for preprocessing, not just those on your training nodes. 
+
+For more details, see the following sections for each framework:
 
 .. tab-set::
 
-    .. tab-item:: PyTorch Dataset and DataLoader
+    .. tab-item:: PyTorch
+
+        **Option 1 (with Ray Data):** 
 
-        **Option 1 (with Ray Data):** Convert your PyTorch Dataset to a Ray Dataset and pass it into the Trainer via  ``datasets`` argument.
-        Inside your ``train_loop_per_worker``, you can access the dataset via :meth:`ray.train.get_dataset_shard`.
-        You can convert this to replace the PyTorch DataLoader via :meth:`ray.data.DataIterator.iter_torch_batches`.
+        1. Convert your PyTorch Dataset to a Ray Dataset and 
+        2. Pass the Ray Dataset into the TorchTrainer via  ``datasets`` argument.
+        3. Inside your ``train_loop_per_worker``, access the sharded dataset via :meth:`ray.train.get_dataset_shard`.
+        4. Create a dataset iterable via :meth:`ray.data.DataIterator.iter_torch_batches`.
 
         For more details, see the :ref:`Migrating from PyTorch Datasets and DataLoaders <migrate_pytorch>`.
 
-        **Option 2 (without Ray Data):** Instantiate the Torch Dataset and DataLoader directly in the ``train_loop_per_worker``.
-        You can use the :meth:`ray.train.torch.prepare_data_loader` utility to set up the DataLoader for distributed training.
+        **Option 2 (with PyTorch DataLoader):** 
+
+        1. Instantiate the Torch Dataset and DataLoader directly in the ``train_loop_per_worker``.
+        2. Use the :meth:`ray.train.torch.prepare_data_loader` utility to set up the DataLoader for distributed training.
 
-    .. tab-item:: LightningDataModule
+    .. tab-item:: Lightning
 
         The ``LightningDataModule`` is created with PyTorch ``Dataset``\s and ``DataLoader``\s. You can apply the same logic here.
 
-    .. tab-item:: Hugging Face Dataset
+    .. tab-item:: Hugging Face
+
+        **Option 1 (with Ray Data):** 
 
-        **Option 1 (with Ray Data):** Convert your Hugging Face Dataset to a Ray Dataset and pass it into the Trainer via the ``datasets`` argument.
-        Inside your ``train_loop_per_worker``, you can access the dataset via :meth:`ray.train.get_dataset_shard`.
+        1. Convert your Hugging Face Dataset to a Ray Dataset. For instructions, see :ref:`Ray Data for Hugging Face <loading_datasets_from_ml_libraries>`.
+        2. Pass the Ray Dataset into the TorchTrainer via the ``datasets`` argument.
+        3. Inside your ``train_loop_per_worker``, access the sharded dataset via :meth:`ray.train.get_dataset_shard`.
+        4. Create a iterable dataset via :meth:`ray.data.DataIterator.iter_torch_batches`. 
+        5. Pass the iterable dataset into ``transformers.Trainer`` during initialization.
 
-        For instructions, see :ref:`Ray Data for Hugging Face <loading_datasets_from_ml_libraries>`.
+        **Option 2 (with HuggingFace Dataset):** 
```

Comment:
nit: I understand why you chose to do this but I'm also a little worried this might be confusing since Option 1 technically does also use Hugging Face Datasets.